devel/cargo/Portfile

```diff
@@ -69,339 +66,349 @@ if { ${os.platform} eq "darwin" && ${os.major} < 10 } {
 # Use MacPorts curl
 # https://doc.rust-lang.org/cargo/commands/cargo-install.html
 # Features of direct dependencies may be enabled with <dep-name>/<feature-name> syntax.
-build.args-append   --features curl-sys/force-system-lib-on-osx
+build.args-append       --features curl-sys/force-system-lib-on-osx
 
 destroot {
-    xinstall ${worksrcpath}/target/[cargo.rust_platform]/release/cargo ${destroot}${prefix}/bin
+    xinstall            ${worksrcpath}/target/[cargo.rust_platform]/release/cargo \
+                        ${destroot}${prefix}/bin
 
-    xinstall -m 0644 \
-        {*}[glob ${worksrcpath}/src/etc/man/*] \
-        ${destroot}${prefix}/share/man/man1
+    xinstall -m 0644    {*}[glob ${worksrcpath}/src/etc/man/*] \
+                        ${destroot}${prefix}/share/man/man1
 
-    xinstall -d ${destroot}${prefix}/share/doc/${subport}
-    xinstall -m 0644 \
-        -W ${worksrcpath} \
-        LICENSE-APACHE LICENSE-MIT LICENSE-THIRD-PARTY README.md \
-        ${destroot}${prefix}/share/doc/${subport}
+    xinstall -d         ${destroot}${prefix}/share/doc/${subport}
+    xinstall -m 0644 -W ${worksrcpath} \
+                        LICENSE-APACHE LICENSE-MIT LICENSE-THIRD-PARTY README.md \
+                        ${destroot}${prefix}/share/doc/${subport}
 
-    xinstall -d ${destroot}${prefix}/etc/bash_completion.d
-    xinstall -m 0644 \
-        ${worksrcpath}/src/etc/cargo.bashcomp.sh \
-        ${destroot}${prefix}/etc/bash_completion.d/${subport}
+    xinstall -d         ${destroot}${prefix}/etc/bash_completion.d
+    xinstall -m 0644    ${worksrcpath}/src/etc/cargo.bashcomp.sh \
+                        ${destroot}${prefix}/etc/bash_completion.d/${subport}
 
-    xinstall -d ${destroot}${prefix}/share/zsh/site-functions
-    xinstall -m 0644 \
-        ${worksrcpath}/src/etc/_cargo \
-        ${destroot}${prefix}/share/zsh/site-functions
+    xinstall -d         ${destroot}${prefix}/share/zsh/site-functions
+    xinstall -m 0644    ${worksrcpath}/src/etc/_cargo \
+                        ${destroot}${prefix}/share/zsh/site-functions
 }
 
 # from cargo2port
```

Comment:
Please add the specific command used to generate these dependencies.

Reply:
I added a little bit to this comment.
For cargo, it is fairly simple.
Just run `cargo2port` in `${worksrcpath}` (as the comments now indicate).